substrate/client/rpc-spec-v2/src/transaction/transaction_broadcast.rs

```diff
@@ -172,6 +209,8 @@ where
 		// The future expected by the executor must be `Future<Output = ()>` instead of
 		// `Future<Output = Result<(), Aborted>>`.
 		let fut = fut.map(move |_| {
+			// Connection space is cleaned when this object is dropped.
+			let _reserved_identifier = reserved_identifier;
```

Comment:
Nit: I'd just call `drop(reserved_identifier)` here to be clearer.

Reply:
This makes sense, thanks!